src/main/java/com/moodybluez/enterprise/dto/Date.java

```diff
@@ -2,9 +2,10 @@
 
 import lombok.Data;
 
+
 public @Data
 class Date {
     private String date;
-    private int dayOfWeekID;
-    private String dayOfWeekDesc;
+    private int dayID;
+    private String dayDesc;
```

Comment:
I made the variable names more general for understandability.

Reply:
There's a difference between day and day of week... I'd keep this as is.  Day of month, day of year, etc...